openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -719,6 +719,15 @@
       // genereal feedback question
       .assessment__rubric__question--feedback {
 
+        .wrapper--input {
+          margin-top: $baseline-v;
+        }
+
+        .question__title__copy {
+          @include margin-left(0);
```

Comment:
Looks like it is used, in some places. I tracked down this instance (https://github.com/edx/edx-ora2/blob/master/openassessment/templates/openassessmentblock/grade/oa_grade_complete.html#L47) on my sandbox, and removing the `margin-left: 10px;` is a bad idea there.

<img width="1238" alt="screen shot 2016-01-21 at 9 54 57 am" src="https://cloud.githubusercontent.com/assets/7373924/12483849/4078b718-c025-11e5-9799-80fff30dec54.png">

Whether we should be using the same class for that text and the `default_feedback_prompt` is probably a discussion we should have at some point, but for now I'm okay with copying the approach used by peer and staff assessments.
https://github.com/edx/edx-ora2/blob/efischer/add_feedback_to_self/openassessment/xblock/static/sass/oa/views/_oa-base.scss#L658
https://github.com/edx/edx-ora2/blob/eff9961015d1164e2b119c7eb9a91e6580740c8d/openassessment/xblock/static/sass/oa/elements/_staff.scss#L230